cmd/kubelet/app/options/options.go

```diff
@@ -57,9 +57,11 @@ func NewKubeletServer() *KubeletServer {
 	}
 }
 
-// AddFlags adds flags for a specific KubeletServer to the specified FlagSet
+// AddFlags adds flags for a specific KubeletServer to the sjpecified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
+	fs.StringVar(&s.PodManifestPath, "config", s.PodManifestPath, "Path to the config file or directory of files")
```

Comment:
Be more descriptive. Path to a single pod manifest file or a directory of pod manifest files